Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalDriver.h

```diff
@@ -95,7 +97,7 @@ typedef NS_ENUM(NSInteger, BTPayPalDriverErrorType) {
 
  On success, you will receive an instance of `BTPayPalAccountNonce`; on failure or user cancelation you will receive an error. If the user cancels out of the flow, the error code will be `BTPayPalDriverErrorTypeCanceled`.
 
- @param request A PayPal request
+ @param request A PayPal checkout request
```

Comment:
```suggestion
 @param request A PayPal Checkout request
```

Reply:
this is a super small nit but I _think_ our dev docs capitalize `Checkout` and `Vault` flows

Reply:
Updated!